src/main/java/com/welovecoding/nbeditorconfig/config/Settings.java

```diff
@@ -5,6 +5,7 @@
 public final class Settings {
 
   public static final String MIME_TYPE = "text/x-editorconfig"; // NOI18N
+  public static final String EXTENSION = "editorconfig"; // NOI18N
```

Comment:
Think of an automatic approach of translating text within sourcecode (or at least separating). I don't think theres something wrong with that comment since the guys from the JDK for example do it the same way. :)